app/src/main/java/com/xinto/opencord/ui/widget/Tree.kt

```diff
@@ -0,0 +1,36 @@
+package com.xinto.opencord.ui.widget
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Path
+import androidx.compose.ui.graphics.drawscope.Stroke
+
+@Composable
+fun WidgetBranchBTR(
```

Comment:
Perhaps `WidgetReplyBranch`? BTR is pretty confusing.

Reply:
I think `WidgetBranchReply` would be a better name, given that we'll also eventually have `WidgetBranchThread`

Reply:
Originally BTR was supposed to mean BottomToRight, but as most likely it'll never be used outside of replies, it'll be a better idea to name it WidgetBranchReply